website/source/docs/job-specification/volume.html.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: "docs"
+page_title: "volume Stanza - Job Specification"
+sidebar_current: "docs-job-specification-volume"
+description: |-
+   The "volume" stanza allows the group to specify that it requires a given volume
+   from the cluster. Nomad will automatically handle ensuring that the volume is
+   available, and mounted into the task.
+---
+
+# `volume` Stanza
+
+<table class="table table-bordered table-striped">
+  <tr>
+    <th width="120">Placement</th>
+    <td>
+      <code>job -> group -> **volume**</code>
+    </td>
+  </tr>
+</table>
+
+The "volume" stanza allows the group to specify that it requires a given volume
+from the cluster. Nomad will automatically handle ensuring that the volume is
+available, and mounted into the task.
+
+The key of the stanza is the name of the volume as it will be exposed to task
```

Comment:
how is the volume name exposed to the task? does the client config `path` define wherein the container the volume will exist as well?

Reply:
ah I see the `task` `volume{}` later down define wherein the container it should go. This isn't clear when reading the docs from top to bottom.